pkg/storagepolicy/sdkstoragepolicy.go

```diff
@@ -128,8 +128,6 @@ func (p *SdkPolicyManager) Create(
 }
 
 // Update Storage policy
-// TODO: Decide whether update storage policy request should update ownership at all
-// We can check if user is admin then only allow update ownership
 func (p *SdkPolicyManager) Update(
```

Comment:
I don't think this is necessary. Update should just take the object given as the new value just as it did before. The caller should do a ReadModifyWrite to update accordingly. This is how Volume ownership, and roles work.

Reply:
Does similar behaviour should be followed for `volumePolicySpec`

Reply:
after discussion with @lpabon in the offline call, SDK should not try to understand changes requested client should do ReadModifyWrite